src/main/scala/wdl4s/cwl/CommandLineTool.scala

```diff
@@ -44,18 +42,18 @@ case class CommandInputEnumSchema(
 
 case class CommandInputArraySchema(
                                     items:
-                                      CwlType :+:
+                                    CwlType :+:
```

Comment:
can you revert this?  Would prefer to have the type indented from the variable declaration